app/assets/javascripts/compute_resources/xenserver/cache_refresh.js

```diff
@@ -0,0 +1,20 @@
+function refreshCache(attribute_name, compute_resource_id, on_success) {
+    tfm.tools.showSpinner();
+    url = "/foreman_xen/cache/refresh";
```

Comment:
Hey Tim,

Thanks for the feedback. I'm not very experienced at rails dev so sorry if this sounds a bit of a simple question, but would you recommend using a new, hidden, element for the data-url?